docs/docs/migration.md

```diff
@@ -4,6 +4,8 @@ If you have previously used the [Multitarget Application Archive Builder](https:
 
 #### Features that handled differently in the Cloud MTA Build Tool
```

Comment:
Features that are handled differently...

Reply:
Updated

Reply:
Thanks